runtime/src/main/scala/akka/grpc/internal/GrpcRequestHelpers.scala

```diff
@@ -24,14 +24,14 @@ object GrpcRequestHelpers {
   def apply[T](
       uri: Uri,
       e: Source[T, NotUsed],
-      eHandler: ActorSystem => PartialFunction[Throwable, Trailers] = GrpcExceptionHandler.defaultMapper)(
-      implicit m: ProtobufSerializer[T],
+      eHandler: ActorSystem => PartialFunction[Throwable, Trailers] = GrpcExceptionHandler.defaultMapper)(implicit
+      m: ProtobufSerializer[T],
       writer: GrpcProtocolWriter,
       system: ClassicActorSystemProvider): HttpRequest =
     request(uri, GrpcEntityHelpers(e, Source.single(GrpcEntityHelpers.trailer(Status.OK)), eHandler))
 
-  private def request[T](uri: Uri, entity: Source[ChunkStreamPart, NotUsed])(
-      implicit writer: GrpcProtocolWriter): HttpRequest = {
+  private def request[T](uri: Uri, entity: Source[ChunkStreamPart, NotUsed])(implicit
+      writer: GrpcProtocolWriter): HttpRequest = {
```

Comment:
I find this dangling `implicit` change confusing.

Reply:
I think I agree. Added `newlines.implicitParamListModifierPrefer = before`